<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1396</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1396</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1396</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dragon Tower</string>
  </property>
  <property name="windowIcon">
   <iconset resource="dane.qrc">
    <normaloff>:/img/jajko_traf.png</normaloff>:/img/jajko_traf.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/smok1.png)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="bet_amount">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="egg_amount">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="balance">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 75)</string>
    </property>
    <property name="text">
     <string>Balance: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_game">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>620</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/new_game.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="mn" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/money.png)</string>
    </property>
   </widget>
   <widget class="QWidget" name="mn_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/jajko_traf.png)</string>
    </property>
   </widget>
   <widget class="QLabel" name="potential">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 75)</string>
    </property>
    <property name="text">
     <string>Potential Win: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="multiplier">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>260</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(150, 150, 75)</string>
    </property>
    <property name="text">
     <string>Multiplier: x0.98</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pole00">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>667</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pole01">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>667</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole02">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>667</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole03">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>667</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole12">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>610</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole13">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>610</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole10">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>610</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole11">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>610</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole22">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>555</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole23">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>555</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole20">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>642</x>
      <y>555</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole30">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>642</x>
      <y>500</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole21">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>555</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole31">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>500</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole32">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>499</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole33">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>499</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole42">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>444</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole43">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>444</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole40">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>643</x>
      <y>444</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole50">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>642</x>
      <y>388</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole41">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>444</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole51">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>732</x>
      <y>388</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole52">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>388</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pole53">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>912</x>
      <y>388</y>
      <width>85</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cash_out">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>620</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/cashout.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="chartarea" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1396</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* styles.qss */

/* Pasek menu */
QMenuBar {
    background-color: #3498db; /* Kolor tła paska menu */
    color: white;              /* Kolor czcionki */
}

/* Elementy menu */
QMenuBar::item {
    background: transparent; /* Przezroczyste tło */
    color: white;             /* Kolor tekstu */
}

/* Kolor tekstu dla wybranych elementów menu */
QMenuBar::item:selected {
    background: #2980b9;      /* Tło przy zaznaczeniu */
    color: yellow;            /* Kolor tekstu przy zaznaczeniu */
}

/* Kolor tekstu po najechaniu (hover) na zakładkę &quot;Creator&quot; */
QMenuBar::item:hover {
    color: white;             /* Kolor czcionki po najechaniu */
}

/* Dostosowanie dla zakładki &quot;Creator&quot; */
QMenuBar::item:focus, QMenuBar::item:selected, QMenuBar::item:hover {
    color: white; /* Kolor czcionki w zakładce &quot;Creator&quot; */
}
QMenuBar::item {
    color: white;
}

QMenuBar::item:selected {
    background-color: #444; /* opcjonalnie, jeśli chcesz zmieniać kolor tła wybranego elementu */
    color: white;
}

QMenuBar::item:pressed {
    background-color: #666; /* opcjonalnie, dla wciśniętego elementu */
    color: white;
}

</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="dane.qrc"/>
 </resources>
 <connections/>
</ui>
